src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -1463,6 +1461,18 @@ private void Model_PropertyChanged(object sender, PropertyChangedEventArgs e)
                 case nameof(DisableTrustWarnings):
                     description = Resources.ResourceManager.GetString(nameof(Res.PreferencesViewTrustWarningHeader), System.Globalization.CultureInfo.InvariantCulture);
                     goto default;
+                // We track these this in two places, one in preference panel,
+                // one where user make such switch in Node AutoComplete UI
+                case nameof(nodeAutocompleteSuggestion):
+                    if (nodeAutocompleteSuggestion == NodeAutocompleteSuggestion.MLRecommendation)
+                        description = nameof(NodeAutocompleteSuggestion.MLRecommendation);
+                    else
+                        description = nameof(NodeAutocompleteSuggestion.ObjectType);
+                    goto default;
+                case nameof(MLRecommendationConfidenceLevel):
+                    // Internal use only, no need to localize for now
+                    description = "Confidence Level";
+                    goto default;
```

Comment:
@RobertGlobant20 I found we had built-in mechanism for property changes tracking on preferences so I moved these here

Reply:
interesting, I haven't seen goto statements in several years